src/main/java/com/hpe/adm/octane/ideplugins/services/nonentity/DownloadScriptService.java

```diff
@@ -36,8 +38,13 @@ public String getTestScriptContent(long testId) {
                     "/workspaces/" + connectionSettings.getWorkspaceId() + "/tests/" + testId + "/script");
             OctaneHttpResponse response = httpClient.execute(request);
             String jsonString = response.getContent();
+            jsonString =  new JsonParser().parse(jsonString).getAsJsonObject().get("script").getAsString();
 
-            return new JsonParser().parse(jsonString).getAsJsonObject().get("script").getAsString();
+            try {
+                return new String(jsonString.getBytes("ISO-8859-1"), "UTF-8");
+            } catch (UnsupportedEncodingException e) {
+                return "Unsupported Encoding";
```

Comment:
do we have any propertie files where this can be kept? if no, it's okay. Also, please create a branch from the version the customer has because this is how we should provide the fix. Alis should test both on a master build and on a specific build from that release

Reply:
make sure on master we test it both on Eclipse and Intellij and on the branch only for the specific ide the customer ahs